src/libraries/System.Runtime.Numerics/tests/BigInteger/op_rightshift.cs

```diff
@@ -29,19 +29,21 @@ public static void BigShiftsTest()
         public static void LargeNegativeBigIntegerShiftTest()
         {
             // Create a very large negative BigInteger
-            BigInteger bigInt = new BigInteger(-1) << int.MaxValue;
-            Assert.Equal(2147483647, bigInt.GetBitLength());
```

Comment:
Should this become a test that validates an exception is thrown?

Reply:
Potentially. There's some general cleanup and additional validation I'd like to do here long term, but I'll get this back up to validate it throws in a follow up PR.